lib/logatron/logatron.rb

```diff
@@ -94,3 +94,5 @@ def msg_id=(id)
     end
   end
 end
+
+require 'logatron/railtie' if defined?(Rails)
```

Comment:
How does this not cause a require loop?  `logatron/railtie` requires this file, too.

Reply:
the `require` keyword is a "once and done" action. Otherwise, all our projects would stack overflow on load!